sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -995,6 +1020,54 @@ Exceptions:
 * Throws synchronously with error code `invalid` if the resulting dependency would
   lead to a cycle. This error is omitted when `property::graph::no_cycle_check` is set.
 
+|
+[source,c++]
+----
+node add_barrier(const std::vector<event> eventList = {});
+----
+
+|Adds a barrier to the graph.
+
+Constraints:
+
+* This member function is only available when the `command_graph` state is
+  `graph_state::modifiable`.
+
+Parameters:
+
+* `eventList` - Zero or more events to wait for.
+  These events can be graph-limited events (from graph recording) 
+  or general sycl events (such as events external to the graph).
```

Comment:
> general sycl events (such as events external to the graph)

"events external to the graph" sounds quite a lot like "External Events". If this is all types of events, i'd just be explicit and enumerate them "These events can be any type of event: graph-limited events (from graph recording), external events, or general sycl events"

Reply:
I've restricted this to external events only as discussed verbally today, and updated the wording here to reflect that.